com.thincast.client.json

```diff
@@ -0,0 +1,337 @@
+                "-DWITH_PCSC:BOOL=ON",
+                "-DWITH_SERVER:BOOL=OFF",
+                "-DWITH_SAMPLE:BOOL=OFF",
+                "-DWITH_CUPS:BOOL=OFF",
+                "-DWITH_OPENH264:BOOL=ON",
+                "-DWITH_OPENH264_LOADING:BOOL=OFF",
+                "-DWITH_FFMPEG:BOOL=OFF",
+                "-DWITH_DSP_FFMPEG:BOOL=OFF",
+                "-DWITH_OSS:BOOL=OFF",
+                "-DWITH_PULSE:BOOL=ON",
+                "-DWITH_ALSA:BOOL=OFF",
+                "-DWITH_OSS:BOOL=OFF",
+                "-DWITH_CAIRO:BOOL=ON",
+                "-DWITH_SWSCALE:BOOL=ON",
+                "-DWITH_WINPR_TOOLS:BOOL=OFF",
+                "-DWITH_WAYLAND:BOOL=OFF",
+                "-DWITH_CLIENT:BOOL=OFF"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/FreeRDP/FreeRDP.git",
+                    "branch": "stable-2.0",
+                    "commit": "c72055d740dcf188d2309fdb5a5c66aee19cca44"
+                }
+            ]
+        },
+        {
+            "name": "libsecret",
+            "config-opts": [
+                "--disable-static",
+                "--disable-gtk-doc",
+                "--disable-manpages"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/libsecret.git",
+                    "tag": "0.20.3"
+                }
+            ]
+        },
+        {
+            "name": "qtkeychain",
+            "buildsystem": "cmake-ninja",
+            "cleanup": [
+                "mkspecs"
+            ],
+            "config-opts": [
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib",
+                "-DLIB_INSTALL_DIR=/app/lib",
+                "-DBUILD_TRANSLATIONS=NO"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/frankosterfeld/qtkeychain/archive/v0.10.0.tar.gz",
+                    "sha256": "5f916cd97843de550467db32d2e10f218b904af5b21cfdfcc7c6425d7dfc3ec2",
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "webaccess",
+            "buildsystem": "simple",
+            "cleanup": [
+                "webaccess.tar.xz"
+            ],
+            "build-commands": [
+                "tar xf webaccess.tar.xz --strip-components=1 -C /app"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "dest-filename": "webaccess.tar.xz",
+                    "url": "https://thincast.com/software/download/artifact/by-hash/45acb93d207f2deaeba0b710547e9750e1d68f385249661104f7146d7e1902e4",
+                    "sha256": "45acb93d207f2deaeba0b710547e9750e1d68f385249661104f7146d7e1902e4",
+                    "only-arches": [
```

Comment:
No need to specify it here (and in other modules) with `flathub.json` limiting the architectures.

Reply:
true, but we are preparing other architectures too (they are just not ready yet).
Can remove this for the time being, of course.

Reply:
I think it's cleaner, but it's your choice.

Reply:
I´d prefer to keep it that way.